### PR TITLE
Doc for differentiating TLS Cert Type & DNS TXT Clarification per CLOUDAUTHZ-5115 &CLOUDAUTHZ-5312

### PR DESCRIPTION
**Pull request by scott-bach**
Description
Added Documentation for differentiating TLS certificate types CLOUDAUTHZ-5115 and Improve/Clarify DNS TXT Record Documentation CLOUDAUTHZ-5312
----

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.
image
image
image

### CODE REVIEW

File: content/authorization/application-registration-prerequisites.md [hunk 1]
```diff
@@ -124,25 +122,39 @@ Please follow one of below approaches to check TLS Certificates for your applica
...
-   1. Supported TLS Certificates types:
+   1. Supported TLS Certificates types (this is usually in a section called "Certificate Policies" as is the case if you view the cert in Firefox)  :
        * Domain Validated (DV),
        * Organization Validated (OV),
        * Extended Validation (EV).
-   1. Supported TLS protocols:
+ ##### Example
+ 
+ Using Firefox to view your TLS certificate may result in a similar Organization Validated certificate as in the example below:
+ [Organization Validated Certificate Type](/images/cert_policies.png)
```
Comment (mbeerman) [inline content/authorization/application-registration-prerequisites.md:143]: This formats a bit oddly - a numbered list interrupted by an outdented aside - but I'm not expert enough with Markdown / Nanoc to be sure there's a better way...

**Comment by dclippard** [comment]
Should we investigate how other browsers (Edge, Chrome, Safari) describe/display certificate type and perhaps call out differences?  


I'm always a little leery of using "cerner.com" as an example (as was done in "new_cert_example.png") because it may confuse the reader.  We're not talking about our cert, we're talking about theirs...  It might be better to choose a neutral third party...

**Comment by scott-bach** [comment]
> Should we investigate how other browsers (Edge, Chrome, Safari) describe/display certificate type and perhaps call out differences?

**I looked at both Chrome and Edge (the only browsers that I have access to) and found no way to discern the cert type. I looked in all areas of the certificate but specifically looked in the "Certificate Policy" and found nothing to show cert type. It seems to be very hard to find this information.**

> I'm always a little leery of using "cerner.com" as an example (as was done in "new_cert_example.png") because it may confuse the reader. We're not talking about our cert, we're talking about theirs... It might be better to choose a neutral third party...

**This was in the previous doc...I just highlighted the Certificate Type in the image. Do you have a suggestion or is Apple or something ok for a net new image for this example? FWIW, we try not to use other applications/vendors in our examples in other documentation...(Help files etc.)**

File: content/authorization/application-registration-prerequisites.md (hunk 1 above)
Comment (scott-bach) [reply content/authorization/application-registration-prerequisites.md:143]: Honestly, neither am I (an expert) and am not sure of a better route. I am making changes in the web ui which does not show the images in the "Preview" of the document. (I will use the local app next time...)

**Review by mbeerman — APPROVED** [review]

**Review by dclippard — APPROVED** [review]
I've reviewed the changes and they look good to me.  Let's get these published.

**Comment by aki-mathur** [comment]
Screenshot 2022-08-26 at 12 51 49 PM

**Pull request merged**